### ISSUE TITLE
Data not collecting

### ISSUE THREAD

**Issue by SuruPa00** [BIDMCDigitalPsychiatry/LAMP-platform#357]
Participant: U5059078716
Although they have been collecting gps / accelerometer (after deleting and reinstalling the app on 8/12/21), screen state data has not been collected in about a month (since around 7/18/21).Although they have been collecting gps / accelerometer (after deleting and reinstalling the app on 8/12/21), screen state data has not been collected in about a month (since around 7/18/21).

Participant: U7842045549 is not collecting data. 
They have a very limited collection of accelerometer data and have not collected any GPS data in the last 2 weeks. Tried deleting and redownloading the app (8/17/21). This did not work.

This lack of data collection seems to be a repeated issue coming up for other participants time to time. Please check on this to see if these are separate instances or if they are tied together and see what can be done to resolve this. Thanks.

**Comment by michaelmenon** [comment]
thank we will check and see if there is any issue in screen data collection and let you know.

**Comment by michaelmenon** [comment]
Ok sure will check on that as well. Just to confirm if the phone is set on Low power mode we will not get the GPS data. os  if we have any issues with GPS data , please check if the device is in low power mode or not. We will check on our end as well.

**Comment by michaelmenon** [comment]
Team checked on this and please see our comments below : 
 
The first participant is iOS and second is an Android user.
 
For iOS, we can investigate 'screen_state' data missing issue and let you know
Another issue is about  'gps' and 'accelerometer' data started collecting only after delete and reinstall. **To investigate this, we would like to know their use case. Like how long they will keep the app without open. so that we can do the similar testing here.**
 
For Android,  the store build is bit old and the new OS version need permission to run in background. And this will be resolved once we have the current Android beta build approved in play store. I think your team is in the process of submitting a video for Google permission issue resolution.
 
-----------------
Also can we implement this https://github.com/BIDMCDigitalPsychiatry/LAMP-platform/issues/125 feature, so that we can get a device's current state like.. 1) permission statuses like location, helathkit, motion etc, 2) battery status, 3) how much data/files pending to sync

**Comment by SuruPa00** [comment]
@michaelmenon  was there any update on why screen duration data was not collecting?

**Comment by SuruPa00** [comment]
We are still running into the issue where screen duration is not being collected. See the list of users who are not collecting screen state data: only a few have collected some but it was in September

`U1077552781
2021-09-17 15:36:47.557000

U1697758293
None

U6808681930
2021-11-09 18:39:32.492000

U7931361682
None

U6393941642
None

U9948995495
None

U7645962928
None

U1289352228
None

U8915149371
None

U4288943539
2021-11-09 18:31:39.291000

U8568896390
2021-11-09 18:30:47.515000

U0010811412
None

U8615089382
None

U1699389494
None

U2180023403
None

U1912527657
None

U7852711409
None

U2032044084
None

U9969216522
None

U2934365190
None

U9389601511
2021-11-09 18:34:45.246000`

**Comment by SuruPa00** [comment]
@michaelmenon can we get an update on this soon?

**Comment by jijopulikkottil** [comment]
@SuruPa00 It seems to be working when I (U3315219392) test here.. Let me check the above user's data.
